rpcs3/Emu/Cell/Modules/cellPngDec.cpp

```diff
@@ -609,7 +615,7 @@ s32 pngDecodeData(PPUThread& ppu, PHandle handle, PStream stream, vm::ptr<u8> da
 		  && (stream->info.colorSpace == CELL_PNGDEC_RGB || stream->info.colorSpace == CELL_PNGDEC_RGBA))
 	{
 		// Tell libpng to convert it to grayscale
-		png_set_rgb_to_gray(stream->png_ptr, PNG_ERROR_ACTION_NONE, PNG_RGB_TO_GRAY_DEFAULT, PNG_RGB_TO_GRAY_DEFAULT);
+		png_set_rgb_to_gray(stream->png_ptr, 1, -1, -1);
```

Comment:
This is how png_set_rgb_to_gray was used prior to version 1.5.7:
http://upstream.rosalinux.ru/compat_reports/libpng/1.5.6_to_1.5.7/abi_compat_report.html